packages/amplication-client/src/Layout/PageContent.scss

```diff
@@ -2,16 +2,15 @@
 
 $side-min-width: 500px;
 $tabs-width: 300px;
+
 .amp-page-content {
   display: flex;
-  flex-direction: row;
-  align-items: stretch;
-  padding: 0 var(--large-spacing);
+  flex: 1;
+  gap: var(--large-spacing);
+  padding: var(--large-spacing);
   background: var(--page-content-background);
   box-sizing: border-box;
-  flex: 1;
   overflow: hidden;
-  height: 100%;
```

Comment:
@EugeneTseitlin why do we need to update the height property to fix the horizontal gap? If there is another reason to remove it, please describe in the details - I am afraid this may have bad impact on the scroll of pages

Reply:
The reason of the bug is redundant styles that conflicted with each other or just were applied not properly. As a solution I've decided to remove everything that is related to positioning (width, height, margin, padding) and make layout from scratch. 
Regarding the height, it should not be used together with flex, flex-grow and flex-shrink are responsible for stretching element relatively to parent container.
Please, see the result on screenshot.